src/test/java/guitests/PlatformExTest.java

```diff
@@ -12,11 +12,6 @@
 
 public class PlatformExTest extends UITest {
 
-    @Override
-    public void launchApp() {
-        FXTestUtils.launchApp(TestUI.class);
```

Comment:
I don't completely remember if that's needed, but it could be that it was mistakenly added. If the test still passes after it's removed, sure.